### PR TITLE
Bugfix/styling

### PR DESCRIPTION
**Pull request by ShahRohan27**
### Changes
- Code clean up and responsiveness issue fixed
- Added isIndeterminate state for Indeterminate checkbox

## Before
Screenshot 2023-02-03 at 10 43 30 AM

Screenshot 2023-02-03 at 10 42 21 AM

## After
Screenshot 2023-02-03 at 10 43 30 AM

Screenshot 2023-02-03 at 10 42 33 AM

### CODE REVIEW

**Comment by RitikAtFynd** [comment]
Looks good to me!

**Comment by Zissan** [comment]
@ShahRohan27 Can you please add before and current images in the PR description.

**Comment by ShahRohan27** [comment]
> @ShahRohan27 Can you please add before and current images in the PR description.

@Zissan Done.

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -152,7 +152,7 @@ const ButtonContent = (props: ButtonProps) => {
           <img
             className="n-btn-spinner"
             style={{ width: "50px" }}
```
Comment (Zissan) [inline src/components/Button/Button.tsx:156]: Should not be having inline style, let's create a class for the image

**Review by Zissan — APPROVED** [review]

**Pull request merged**